mlflow/store/dbfs_fuse_artifact_repo.py

```diff
@@ -0,0 +1,11 @@
+from mlflow.store.local_artifact_repo import LocalArtifactRepository
+
+
+class DbfsFuseArtifactRepository(LocalArtifactRepository):
```

Comment:
One side-effect of inheriting from the `LocalArtifactRepository` (used to write artifacts to the local filesystem, e.g. the ./mlruns directory) is that the DBFS FUSE artifact repository will overwrite existing files, while the REST-backed repository will not. I don't think this behavior difference is particularly significant, but we could choose to make the two consistent.